test/integration/targets/pip/tasks/pip.yml

```diff
@@ -499,6 +499,7 @@
       name: distribute
       virtualenv: "{{ output_dir }}/pipenv"
       state: present
+      extra_args: "-c {{ remote_constraints }}"
```

Comment:
It looks like the `pip` version on centos6 is too old to support `-c`. Maybe it's better to do it as in https://github.com/ansible/ansible/pull/66405/files#diff-19580f9458658fce25ba615e17cce547R499-R502 ?